.github/workflows/build-and-push-docker.yml

```diff
@@ -0,0 +1,39 @@
+name: Docker Build and Push
+
+# Controls when the action will run. 
+# Triggers the workflow for a push to dev
+# with changes in Dockerfile.
+on:
+  push:
```

Comment:
Does GHA have an `on merge`?

Reply:
checking

Reply:
I guess this is how we would setup the action on merge  as per this blog post https://brennerm.github.io/posts/trigger-github-actions-on-pr-close.html

implemented the above in this commit https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/14/commits/55e5728c6fd11a17b5ff0df7486e5b6575bed9f2

Reply:
Interesting. Good to know.